Misc/NEWS.d/next/Core and Builtins/2021-05-25-18-20-10.bpo-44232.DMcCCf.rst

```diff
@@ -0,0 +1,3 @@
+Fix a regression in :func:`type` when a metaclass raises an exception. The C
+function :c:func:`type_new` must properly report the exception when a metaclass
+constructor raises an exception and the winner class is not the metaclass.
```

Comment:
I'm not sure if it's worth mentioning in the blurb that this bug only affected people using debug builds. Release build users shouldnt have noticed anything.

Reply:
I think we should mention it as people use debug build outside Cpython development quite a lot

Reply:
I didn't expect anyone to use a Python debug build until I saw https://bugs.python.org/issue44232

IMO it's worth it to document the bugfix since it impacts Django, but likely many more projects.

Reply:
Wow I need to stop sending messages late at night ;-). I'm really sorry this concern got lost in translation. I wasn't suggesting we get rid of the blurb. I was suggesting we add a line somewhere that says this only affects debug builds and not release ones. Previously I did that for another AssertionError crash in 3.9 - 3.10 because I was afraid people would read the blurb and think their Python is affected.

https://github.com/python/cpython/commit/6a7fb9d31bce8590e30c44458d1fc1da4539743d

Anyways, I think it's fine if you dont want to. Thanks for fixing this :).